mentoring/public/themes/lms.css

```diff
@@ -0,0 +1,18 @@
+.themed-xblock.mentoring .checkmark-incorrect {
+    color: #c1373f;
```

Comment:
Some styles like `.mentoring .checkmark-incorrect { color: }` are now specified in both `mentoring.css` and `themes/lms.css`. Perhaps we can remove the colors from mentoring.css? Or is it important to keep that as a fallback?